src/ui/hexedit.cc

```diff
@@ -643,10 +660,32 @@ QString HexEdit::addressAsText(qint64 pos) {
 }
 
 QString HexEdit::asciiRepresentationFromByte(uint64_t byte_val) {
-  if (byte_val >= 0x20 && byte_val < 0x7f) {
-    return QChar::fromLatin1(byte_val);
+  static QTextCodec* windows1250 = QTextCodec::codecForName("windows-1250");
+
+  if (unprintablesModeString.compare("windows-1250") == 0) {
+    // 0x85 is NEL, 0xa0 is nbsp in unicode, but we want to display them
+    // 0x83, ..., 0x98 are undefined in used codeing
+    if (byte_val != 0x85 && byte_val != 0xa0 &&
+        (QChar(static_cast<uint>(byte_val)).isSpace() ||
+         QChar(static_cast<uint>(byte_val)).isNull() || byte_val == 0x83 ||
+         byte_val == 0x88 || byte_val == 0x90 || byte_val == 0x98)) {
+      return QChar(static_cast<uint>(' '));
+    }
+
+    // printable ascii chars
+    if (byte_val >= 0x20 && byte_val < 0x7f) {
+      return QChar::fromLatin1(byte_val);
+    }
+
+    // ascii-unprintable chars
+    char a = (static_cast<char>(byte_val));
```

Comment:
This conversion is implementation-defined for `byte_val` >= 0x80, please wrap the higher values by hand (or create a helper function in `veles::utils::`).
And yes, this seems to be a general problem with Qt APIs - they use char* everywhere for byte arrays.

Reply:
Done.